### DISCUSSION TITLE
Interesting, PPOCR-V3 takes 6+s/png on a CPU i386 processor, should it be much faster as claimed?

### DISCUSSION THREAD

**Discussion by alexucb** [PaddlePaddle/PaddleOCR#8141] [Q&A]
Hi, 

First, thanks for continuing the improvement work on PPOCR. 

Recently I tested ppocr-v3 on my laptop, looks like it takes 6+s/per page for text detection, which is far away from the benchmark results. I was wondering whether there are some parameters i had messed up. would any of you mind helping me to understand why and guide me on how to achieve text detection in hundreds of milliseconds as reported benchmark result on cpu. Thank you in advance!

My laptop configuration and script are as following:
<img width="1885" alt="image" src="https://user-images.githubusercontent.com/29640620/198369015-41f2030a-2148-412f-9637-361a69bc3cf1.png">

**Comment by LDOUBLEV** [comment]
Please set enable_mkldnn=True when initializing PaddleOCR().

When predicting the first image, the model needs to be initialized, which will take a certain amount of time. After predicting the second picture, the inference speed will be much faster than the first inference.

**Reply by alexucb** [reply]
sorry 还是不行
<img width="1837" alt="image" src="https://user-images.githubusercontent.com/29640620/200386680-ceb4a3e5-82eb-4c9d-8d07-9f9dcdc8db6c.png">

**Reply by manoadamro** [reply]
@alexucb I'm getting the same. AFAIK mkldnn no longer exists, I think It's now called onednn.
I have it installed on my machine and have even tried it with conda. Did you manage to solve this?
